playbooks/roles/mysql/defaults/main.yml

```diff
@@ -39,3 +39,19 @@ MYSQL_EXPORTER_CONSUL_SERVICE:
       - monitor-https
     port: 19104
     enable_tag_override: true
+
+# MySQL Binaries for pinning to version 5.6.35 #########################################
+
+MYSQL_DEBIAN_SNAPSHOT_URL: http://snapshot.debian.org/archive/debian/20170120T212942Z/pool/main/m/mysql-5.6
```

Comment:
@farhaanbukhsh, @giovannicimolin, is this a supported and maintained debian package? If not, we should just use the binaries from MySQL directly.

Reply:
These are signed and supported binaries from Debian. @lgp171188

Reply:
@farhaanbukhsh, signed by Debian doesn't matter. "Supported" means being a supported package in a supported release, which is "Debian 10" and "Debian 9" LTS (but may not have a supported MySQL package).

Reply:
@giovannicimolin ^

Reply:
Okay that does make sense, should we schedule a task to do that?

Reply:
@lgp171188 I could see only https://downloads.mysql.com/archives/community/?version=5.6.44 5.6.44 which has Ubuntu supported binaries. Is there any other place where we can get deb packages for 5.6.50?

Reply:
@farhaanbukhsh, the task description says

> Update https://github.com/open-craft/ansible-playbooks/tree/master/playbooks/roles/mysql so that MySQL is installed from the binaries.

It doesn't say "deb" packages, though they would be very convenient and easy to use. That is also why I was referring to installing from the binary tarball release from the MySQL site.

I'll let @giovannicimolin comment on this and decide on the next steps, if any.

Reply:
@lgp171188 ,

There is more clarification in the comment https://tasks.opencraft.com/browse/BB-3036?focusedCommentId=169392&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-169392 

Also I think it was a call because of MySQL 5.6 reaching EOL and a lot of services migrating and using 5.7 even for Edx. 😄 
 So we decided to have a workable solution till then.

Reply:
@lgp171188 I think I've decided for the approach because of two reasons:

1. MySQL 5.6 EOL: which would force us to upgrade in the near future.
2. Koa and upcoming releases with update MySQL version compatibility.

So it didn't make sense to spend a lot of hours given we'll have to rework our playbooks to deploy the newest version soon. I just wanted the playbooks to work in case we need to do a emergency redeployment until we upgrade the service.